es6/assignment.js

```diff
@@ -1,96 +1,97 @@
-import { PlanOutOpRandom } from "./ops/random";
 import { shallowCopy, forEach, hasKey } from "./lib/utils";
 
-class Assignment {
-  constructor(experimentSalt, overrides) {
-    if (!overrides) {
-      overrides = {};
+export default function provideAssignement(Random) {
```

Comment:
the diff shows a lot of changes but just confirming that the only changes here are basically wrapping the export as the `provideAssignment` function instead of just exporting the class?